kubernetes/data_source_kubernetes_endpoints.go

```diff
@@ -0,0 +1,35 @@
+package kubernetes
```

Comment:
Please rename this file with the `apiVersion` suffix -- `data_source_kubernetes_endpoints_v1.go` and the same for the related test file.